src/delimiter.js

```diff
@@ -323,7 +373,12 @@ const makeStackedDelim = function(delim, heightTotal, center, options, mode,
 const vbPad = 80;   // padding above the surd, measured inside the viewBox.
 const emPad = 0.08; // padding, in ems, measured in the document.
 
-const sqrtSvg = function(sqrtName, height, viewBoxHeight, options) {
+const sqrtSvg = function(
+    sqrtName: string,
```

Comment:
We should be able to scope this to specific strings, e.g. "sqrtTall", etc.

Reply:
`sqrtName` can take one of five values:, "sqrtMain", "sqrtSize1", "sqrtSize2", "sqrtSize3", "sqrtSize4", "sqrtTall"

Reply:
It's called with `sqrtSvg("sqrtSize" + delim.size, ...)` which doesn't automatically satisfy `"sqrtSize1" | "sqrtSize2" | "sqrtSize3"`. Since this is a local function only called within this file, I'm not sure it's worth having to override the type-checker to accept this code.

Reply:
> a local function only called within this file, 

I agree that not much is gained by another restriction.